src/Block/BlockServiceManager.php

```diff
@@ -191,11 +192,12 @@ public function validate(ErrorElement $errorElement, BlockInterface $block): voi
 
     /**
      * @throws \RuntimeException
+     * @throws BlockNotFoundException
      */
     private function load(string $type): BlockServiceInterface
     {
         if (!$this->has($type)) {
-            throw new \RuntimeException(sprintf('The block service `%s` does not exist', $type));
+            throw new BlockNotFoundException(sprintf('The block service `%s` does not exist', $type));
```

Comment:
This is not really correct right? Block is found, what is not found is the BlockService, maybe it needs a new Exception class? like BlockServiceNotFound?

Reply:
Yes indeed, might be great to introduce BlockServiceNotFoundException

Reply:
Done in my last commit, Wdyt ?